### ISSUE TITLE
Add support for one way platforms

### ISSUE THREAD

**Issue by britzl** [britzl/platypus#7]
Perhaps add a way to configure collision groups so that they will ignore collisions in certain directions. 

```
platypus.UP = x01
platypus.LEFT = x02
platypus.RIGHT = x04
platypus.DOWN = x08

platypus.create({
	collisions = {
		ground = {
			[hash("ground")] = platypus.UP + platypus.LEFT + platypus.RIGHT + platypus.DOWN,
			[hash("onewayplatform")] = platypus.LEFT + platypus.RIGHT + platypus.DOWN,
		}
	}
})
```

The config decides how to deal with contact_point_response and ray_cast_response messages.

**Comment by britzl** [comment]
BTW, `config.collisions.ground` should probably be renamed to `config.collisions.groups`

**Comment by britzl** [comment]
Fixed in 3.1.0
